app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
Both Ron and I checked that the 2.jpg files itself don't contain header information such as MIME type, do they?

- They do not. We set them based on the extracted mimetype from JHOVE (and FITS for Horton).

If the derivative is fine that are not corrupted(like the example in ticket), re-creating derivative by image-magic would't fix the problem

- It might, actually. Because those derivatives were likely created on a much older version of imagemagick. 

but we should try to re-running JHOVE to fix the metadata in triplestore.
Otherwise, both need to re-generate.

- 👍 (for the same reason as above, many of them were likely run against an older version of JHOVE)